molecule/zabbix_proxy_psk/molecule.yml

```diff
@@ -0,0 +1,91 @@
+---
+dependency:
+  name: galaxy
+  enabled: true
+  options:
+    requirements-file: molecule/default/collections.yml
+
+driver:
+  name: default
+
+platforms:
+  - name: zabbix-proxy-psk-${MY_MOLECULE_VERSION:-v70}-${MY_MOLECULE_DATABASE:-mysql}-${MY_MOLECULE_CONTAINER:-rockylinux8}
+    image: geerlingguy/docker-${MY_MOLECULE_IMAGE:-rockylinux8}-ansible:latest
+    privileged: true
+    pre_build_image: true
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    networks:
+      - name: zabbix
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+    cgroupns_mode: host
+    zabbix_proxy_version: ${MY_MOLECULE_VERSION:-v70}
+    groups:
+      - ${MY_MOLECULE_DATABASE:-mysql}
+      - ${MY_MOLECULE_VERSION:-v70}
+
+provisioner:
+  name: ansible
+  playbooks:
+    create: ../zabbix_proxy/create.docker.yml
+    destroy: ../zabbix_proxy/destroy.docker.yml
```

Comment:
add
```
converge: ../zabbix_proxy/converge.yml
```
to reuse just like create/destroy.

Reply:
Fixed in https://github.com/BGmot/community.zabbix/commit/efcc802a88fe8fd2573719db1e6d16c895c16032